src/modules/ekf2/EKF/gps_control.cpp

```diff
@@ -80,9 +79,15 @@ void Ekf::controlGpsFusion()
 						if (isYawFailure()
 						    && _control_status.flags.in_air
 						    && !was_gps_signal_lost
-						    && _ekfgsf_yaw_reset_count < _params.EKFGSF_reset_count_limit) {
-
-							_do_ekfgsf_yaw_reset = true;
+						    && _ekfgsf_yaw_reset_count < _params.EKFGSF_reset_count_limit
+						    && isTimedOut(_ekfgsf_yaw_reset_time, 5000000)) {
+							/* The yaw reset to the EKF-GSF estimate can be requested externally at any time during flight.
```

Comment:
You removed the public setter in this PR, right?

Reply:
Comment updated.